third_party/move/move-vm/types/src/values/values_impl.rs

```diff
@@ -230,12 +258,16 @@ impl Container {
             Self::VecU256(r) => r.borrow().len(),
             Self::VecBool(r) => r.borrow().len(),
             Self::VecAddress(r) => r.borrow().len(),
+
+            Self::Locals(r) => r.borrow().len(),
```

Comment:
why this change?

Reply:
Matches were done differently and wanted to make them uniform, and structured in order